src/core/vectortile/qgsvectortilelayerrenderer.cpp

```diff
@@ -48,26 +49,29 @@ bool QgsVectorTileLayerRenderer::render()
   QElapsedTimer tTotal;
   tTotal.start();
 
-  qDebug() << "MVT rend" << ctx.extent().toString( -1 );
+  QgsDebugMsg( QStringLiteral( "Vector tiles rendering extent: " ) + ctx.extent().toString( -1 ) );
```

Comment:
Can you bump these to level 2 or more? We should only ever use QgsDebugMsg() for reporting errors.